apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -63,3 +80,35 @@ export const WarningLabel = ({children}) => (
 WarningLabel.propTypes = {
   children: React.PropTypes.node
 };
+
+/**
+ * Given a search query, generate a results list of animationProps objects that
+ * can be displayed and used to add an animation to the project.
+ * @param {string} searchQuery - text entered by the user to find an animation
+ * @return {Array.<SerializedAnimationProps>} - roughly
+ */
+function searchAnimations(searchQuery) {
+  // Make sure to generate the search regex in advance, only once.
+  // Search is case-insensitive
+  // Match any word boundary or underscore followed by the search query.
+  // Example: searchQuery "bar"
+  //   Will match: "barbell", "foo-bar", "foo_bar" or "foo bar"
+  //   Will not match: "foobar", "ubar"
```

Comment:
I think this is what Caley was getting at but just in case not — it would be awesome to somehow save out failed searches (once user backtracks when no results?) to see what graphics students wish we had (& potential mis-searches). I know of at least one coding website that has done that as a way to source graphic additions and kids loved it (really makes it feel like the website offers something for everything).